pages/contribute/README.md

```diff
@@ -0,0 +1,50 @@
+---
+navigation_title: "Contribute"
+position: 5
+changed: "2018-06-15"
+---
+
+# Contribute to this guide
+
+**This guide is the result of tight collaboration between accessibility experts and web developers. Its content by far is not exhaustive yet: there remain a lot of topics to talk about, and due to the fast evolution of web technologies, there certainly will always be.**
+
+We welcome everyone who wants to join our initiative, be it with adding new contents, fixing typos or bugs, etc.
+
+## What you can do
+
+### Discuss about accessibility and the guide
+
+Come and [join our Slack team] or [create an issue on GitHub].
+
+### Contribute code
+
+Improve the guide by fixing bugs or adding code examples. The code is completely open-source and can be found on GitHub under [Access4all/adg (GitHub.com)].
+
+We are happy about issues and pull requests. Please refer to our contribution guidelines: [Access4all/adg/CONTRIBUTING.md (GitHub.com)].
+
+Let us make the Internet a more accessible place together!
+
+### Share your knowledge
+
+Help to improve the content of the guide by adding new chapters or improving existing chapters. [Join our Slack team] to discuss about your contribution and how to send it in, create an issue or a pull request on our GitHub project [Access4all/adg (GitHub.com)] to show us your proposal directly.
+
+Also, we appreciate if you found mistakes or outdated sections in the guide. Please let us know.
+
+### Meet at the Accessibility Developer Hackday
+
+We organise frequent hackdays to discuss and plan the state of the Accessibility Developer Guide. [Join our Meetup.com group] to get notified about upcoming events.
```

Comment:
Let's add something like "and implement new content and examples" together.

And: "Or we take an in-depth look at an interesting accessibility question together."